### PR TITLE
new target added L-TEK FF1705

### PR DESCRIPTION
**Pull request by gorazdko**
* compiling and programming ok
* board id set to 8080 -> confirmed from mbedos
* test info added.
* rebased and squashed

### CODE REVIEW

**Comment by gorazdko** [comment]
Couldnt build tests though, apparently some error in some other target?


(env) C:\Users\Gorazd\Desktop\LORA-FF1705\DAPLink\test>python run_test.py --help
usage: run_test.py [-h] [--targetdir TARGETDIR] [--user USER]
                   [--password PASSWORD] [--firmwaredir FIRMWAREDIR]
                   [--firmware {k20dx_frdmkl02z_if,lpc11u35_sscity_if,k20dx_frdmkl27z_if,lpc11u35_wizwiki_w7500_eco_if,kl26z_nina_b1_if,k20dx_frdmk66f_if,k20dx_frdmk20dx_if,lpc11u35_ssci1114_if,lpc11u35_arm_watch_nrf51_if,lpc11u35_nz32_sc151_if,k20dx_xdot_l151_if,k20dx_frdmke15z_if,sam3u2c_ncs36510rf_if,lpc11u35_arm_watch_stm32f411_if,lpc11u35_lpc4088dm_if,lpc11u35_gr_peach_if,k20dx_frdmkl26z_if,k20dx_frdmk28f_if,k20dx_frdmkl46z_if,lpc11u35_lpc824xpresso_if,k20dx_frdmk82f_if,lpc11u35_wizwiki_w7500p_if,kl26z_microbit_if,lpc11u35_hrm1017_if,lpc11u35_lpc4088qsb_if,lpc11u35_6lowpan_borderrouterhat_if,lpc11u35_cocorico_if,lpc11u35_6lowpan_borderrouterusb_if,lpc11u35_archmax_if,lpc11u35_wizwiki_w7500_if,lpc11u35_archpro_if,lpc11u35_ssci824_if,k20dx_frdmkl25z_if,lpc11u35_c027_if,lpc11u35_arm_watch_efm32_if,k20dx_frdmkl05z_if,k20dx_frdmkl43z_if,lpc11u35_blueninja_if,k20dx_rblnano_if,lpc11u35_FF1705_L151_if,k20dx_rbl_if,lpc11u35_ssci_chibi_if,k20dx_frdmkl82z_if,lpc11u35_tiny_if,lpc11u35_archlink_if,lpc11u35_lpc812xpresso_if,k20dx_frdmk22f_if,lpc11u35_6lowpan_borderrouterethernet_if,k20dx_frdmkw24d_if,k20dx_frdmkl28z_if,k20dx_twrke18f_if,k20dx_twrkl28z72m_if,sam3u2c_mkit_dk_dongle_nrf5x_if,k20dx_hvpke18f_if,lpc11u35_archble_if,k20dx_frdmk64f_if}]
                   [--logdir LOGDIR] [--noloadif] [--notestendpt] [--loadbl]
                   [--testdl] [--testfirst]
                   [--verbose {Minimal,Normal,Verbose,All}] [--dryrun]
                   [--force]

DAPLink validation and testing tool

optional arguments:
  -h, --help            show this help message and exit
  --targetdir TARGETDIR
                        Directory with pre-built target test images.
  --user USER           MBED username (required for compile-api)
  --password PASSWORD   MBED password (required for compile-api)
  --firmwaredir FIRMWAREDIR
                        Directory with firmware images to test
  --firmware {k20dx_frdmkl02z_if,lpc11u35_sscity_if,k20dx_frdmkl27z_if,lpc11u35_wizwiki_w7500_eco_if,kl26z_nina_b1_if,k20dx_frdmk66f_if,k20dx_frdmk20dx_if,lpc11u35_ssci1114_if,lpc11u35_arm_watch_nrf51_if,lpc11u35_nz32_sc151_if,k20dx_xdot_l151_if,k20dx_frdmke15z_if,sam3u2c_ncs36510rf_if,lpc11u35_arm_watch_stm32f411_if,lpc11u35_lpc4088dm_if,lpc11u35_gr_peach_if,k20dx_frdmkl26z_if,k20dx_frdmk28f_if,k20dx_frdmkl46z_if,lpc11u35_lpc824xpresso_if,k20dx_frdmk82f_if,lpc11u35_wizwiki_w7500p_if,kl26z_microbit_if,lpc11u35_hrm1017_if,lpc11u35_lpc4088qsb_if,lpc11u35_6lowpan_borderrouterhat_if,lpc11u35_cocorico_if,lpc11u35_6lowpan_borderrouterusb_if,lpc11u35_archmax_if,lpc11u35_wizwiki_w7500_if,lpc11u35_archpro_if,lpc11u35_ssci824_if,k20dx_frdmkl25z_if,lpc11u35_c027_if,lpc11u35_arm_watch_efm32_if,k20dx_frdmkl05z_if,k20dx_frdmkl43z_if,lpc11u35_blueninja_if,k20dx_rblnano_if,lpc11u35_FF1705_L151_if,k20dx_rbl_if,lpc11u35_ssci_chibi_if,k20dx_frdmkl82z_if,lpc11u35_tiny_if,lpc11u35_archlink_if,lpc11u35_lpc812xpresso_if,k20dx_frdmk22f_if,lpc11u35_6lowpan_borderrouterethernet_if,k20dx_frdmkw24d_if,k20dx_frdmkl28z_if,k20dx_twrke18f_if,k20dx_twrkl28z72m_if,sam3u2c_mkit_dk_dongle_nrf5x_if,k20dx_hvpke18f_if,lpc11u35_archble_if,k20dx_frdmk64f_if}
                        Firmware to test
  --logdir LOGDIR       Directory to log test results to
  --noloadif            Skip load step for interface.
  --notestendpt         Dont test the interface USB endpoints.
  --loadbl              Load bootloader before test.
  --testdl              Run DAPLink specific tests. The DAPLink test tests
                        bootloader updates so usewith caution
  --testfirst           If multiple boards of the same type are found only
                        test the first one.
  --verbose {Minimal,Normal,Verbose,All}
                        Verbose output
  --dryrun              Print info on configurations but dont actually run
                        tests.
  --force               Try to run tests even if there are problems. Delete
                        logs from previous run.

(env) C:\Users\Gorazd\Desktop\LORA-FF1705\DAPLink\test>python run_test.py --user gorazdko --password ***** --firmware lpc11u35_FF1705_L151_if
SubTest: Building target KL25Z
Starting remote build
Build took 5.50200009346 seconds
SubTest: Building target FRDM-KL05Z
Starting remote build
Build took 4.99000000954 seconds
SubTest: Building target FRDM-KL46Z
Starting remote build
Build took 5.00800013542 seconds
SubTest: Building target FRDM-K20D50M
Starting remote build
Build took 4.93299984932 seconds
SubTest: Building target FRDM-K22F
Starting remote build
Build took 5.09099984169 seconds
SubTest: Building target FRDM-K64F
Starting remote build
Build took 5.11199998856 seconds
SubTest: Building target LPCXpresso824-MAX
Starting remote build
Build took 4.92500019073 seconds
SubTest: Building target mbed-HRM1017
Starting remote build
Build took 8.42200016975 seconds
SubTest: Building target Switch-Science-mbed-LPC824
Starting remote build
Build took 4.77200007439 seconds
SubTest: Building target Switch-Science-mbed-TY51822r3
Starting remote build
Build took 11.2140002251 seconds
SubTest: Building target NXP-LPC800-MAX
Starting remote build
Build took 4.89800000191 seconds
SubTest: Building target EA-LPC4088
Starting remote build
Build took 4.91599988937 seconds
SubTest: Building target EA-LPC4088-Display-Module
Starting remote build
Build took 4.93799996376 seconds
SubTest: Building target Nordic-nRF51822
Starting remote build
Build took 7.79399991035 seconds
SubTest: Building target RedBearLab-nRF51822
Starting remote build
Build took 7.86999988556 seconds
SubTest: Building target RedBearLab-BLE-Nano
Starting remote build
Build took 8.07299995422 seconds
SubTest: Building target Nordic-nRF51-DK
Starting remote build
Build took 10.9900000095 seconds
SubTest: Building target Nordic-nRF52-DK
Starting remote build
Build took 11.2430000305 seconds
SubTest: Building target LPC1114FN28
Starting remote build
Build took 4.69400000572 seconds
SubTest: Building target Nordic-nRF51-Dongle
Starting remote build
Build took 11.143999815 seconds
SubTest: Building target u-blox-C027
Starting remote build
Build took 4.88800001144 seconds
SubTest: Building target ncs36510
Starting remote build
Build took 5.01900005341 seconds
SubTest: Building target Seeeduino-Arch-Pro
Starting remote build
Build took 4.96799993515 seconds
SubTest: Building target Seeed-Arch-BLE
Starting remote build
Build took 7.79799985886 seconds
SubTest: Building target Seeed-Arch-Max
Starting remote build
Build took 8.07700014114 seconds
SubTest: Building target Seeed-Tiny-BLE
Starting remote build
Build took 8.94200015068 seconds
SubTest: Building target Seeed-Arch-Link
Starting remote build
Build took 8.32500004768 seconds
SubTest: Building target Microbit
Starting remote build
Build took 7.88300013542 seconds
SubTest: Building target mbed-6LoWPAN-Border-Router-HAT
Starting remote build
Build took 5.14699983597 seconds
SubTest: Building target mbed-6LoWPAN-Border-Router-USB
Starting remote build
Build took 5.22399997711 seconds
SubTest: Building target mbed-6LoWPAN-Border-Router-ETHERNET
Starting remote build
Build took 5.01999998093 seconds
SubTest: Building target Renesas-GR-PEACH
Starting remote build
Build took 5.45799994469 seconds
SubTest: Building target NZ32-SC151
Starting remote build
Traceback (most recent call last):
  File "run_test.py", line 701, in <module>
    main()
  File "run_test.py", line 587, in main
    build_target_bundle(target_dir, args.user, args.password, test_info)
  File "C:\Users\Gorazd\Desktop\LORA-FF1705\DAPLink\test\target.py", line 91, in build_target_bundle
    directory)
  File "C:\Users\Gorazd\Desktop\LORA-FF1705\DAPLink\test\mbedapi.py", line 133, in build_repo
    return destination
UnboundLocalError: local variable 'destination' referenced before assignment

(env) C:\Users\Gorazd\Desktop\LORA-FF1705\DAPLink\test>

File: test/info.py [hunk 1]
```diff
@@ -64,6 +64,7 @@
     ("lpc11u35_lpc4088dm_if",                       False,      0x0000,     "bin"       ),
     ("lpc11u35_lpc4088qsb_if",                      False,      0x0000,     "bin"       ),
     ('lpc11u35_ssci_chibi_if',                      False,      0x0000,     "bin"       ),
+	('lpc11u35_FF1705_L151_if',                     False,      0x0000,     "bin"       ),
```
Comment (c1728p9) [inline test/info.py]: This should be spaces rather than tabs. Same for the line below.

**Comment by cederom** [comment]
Test fail reason: ```Bad project name "lpc11u35_FF1705_L151_if"```

**Comment by cederom** [comment]
Do we add this to STAGE @c1728p9 ?

**Pull request merged**